.github/workflows/presubmit.yml

```diff
@@ -0,0 +1,54 @@
+name: presubmit
+
+on: pull_request
+
+jobs:
+  lint-js:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Install ESLint with Google Config
+      run: npm install eslint@^7.3.1 eslint-config-google@^0.14.0
+    - name: Check
+      run: npx eslint src
+
+  lint-java:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Check
+      run: mvn spotless:check
```

Comment:
You need add spotless plugin:
 <!-- Provides `mvn spotless:check` and `mvn spotless:apply`. -->
            <plugin>
                <groupId>com.diffplug.spotless</groupId>
                <artifactId>spotless-maven-plugin</artifactId>
                <version>1.31.3</version>
                <configuration>
                    <java>
                        <googleJavaFormat>
                            <version>${google-java-format.version}</version>
                        </googleJavaFormat>
                    </java>
                </configuration>
            </plugin>